lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -180,6 +181,12 @@ swapNodes = (targetBody, existingNodes, options) ->
         onNodeRemoved(existingNode)
   return
 
+removeNodes = (body, selector) ->
+  for node in findNodes(body, selector)
+    node.remove()
+    onNodeRemoved(node)
```

Comment:
Don't need this. `onNodeRemoved` was already called on the element or one of its parents when a new page was loaded.